activestorage/app/models/active_storage/variation.rb

```diff
@@ -88,12 +88,13 @@ def mini_magick_transform(file, format)
 
       transformations.each do |name, argument_or_subtransformations|
         image.mogrify do |command|
+
           if name.to_s == "combine_options"
             argument_or_subtransformations.each do |subtransformation_name, subtransformation_argument|
-              pass_transform_argument(command, subtransformation_name, subtransformation_argument)
+              pass_transform_argument(command, subtransformation_name, subtransformation_argument) if subtransformation_argument.present?
             end
           else
-            pass_transform_argument(command, name, argument_or_subtransformations)
+            pass_transform_argument(command, name, argument_or_subtransformations) if argument_or_subtransformations.present?
```

Comment:
I’d prefer to push this condition down to `pass_transform_argument` rather than repeat it at each callsite:

```diff
   def pass_transform_argument(command, method, argument)
     if eligible_argument?(argument)
       command.public_send(method, argument)
-    else
+    elsif argument
       command.public_send(method)
     end
   end
```